packages/electrode-archetype-react-app-dev/config/webpack/add-dll-references.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
       }));
     }
   } catch (err) {
-    //eslint-disable-line no-empty
+    console.log("add-dll-references failed", err);
```

Comment:
`console.error` ? this won't abort execution, but it will output like a fatal error (and get colourised red)